labfunctions/conf/jtemplates.py

```diff
@@ -9,19 +9,18 @@ def get_package_dir(pkg):
     return spec.submodule_search_locations[0]
 
 
-env = Environment(
-    loader=FileSystemLoader(f"{get_package_dir('labfunctions')}/conf/templates"),
-    autoescape=select_autoescape(),
-)
-
-
-def render(filename, *args, **kwargs):
+def render(filename, templates_dir=None, *args, **kwargs):
+    path = f"{get_package_dir('labfunctions')}/conf/templates" if templates_dir is None else get_package_dir(templates_dir)
```

Comment:
I'm not sure if this will work. If a value for templates_dir is provided then **get_package_dir()** should be avoided, because the user could be provided a fullpath and not a relative path to the labfunctions package.

Reply:
It's already fixed in the latter commit ([see](https://github.com/labfunctions/labfunctions/pull/1/commits/6f92602513a0217e832127702aadeade2d77e442))